apps/charterafrica/src/payload/blocks/MediaBlock.js

```diff
@@ -0,0 +1,23 @@
+import blockFields from "../fields/blockFields";
+
+const MediaBlock = {
+  slug: "mediaBlock",
+  fields: [
+    blockFields({
+      name: "mediaBlockFields",
+      fields: [
+        {
+          name: "media",
```

Comment:
```suggestion
          name: "image",
```
This way, FE will know this block is just for image (and not audio or video or PDF file, etc.)